### PR TITLE
Nfqueue Balance attempt 2

### PR DESCRIPTION
**Pull request by amitlimaye**
#### Description
*Changes proposed in this pull request.*

#### Test plan
*Outline the test plan used to test this change before merging it.*

> Fixes #.

### CODE REVIEW

File: controller/internal/supervisor/iptablesctrl/iptables.go [hunk 1]
```diff
@@ -275,11 +277,12 @@ func (i *iptables) DeleteRules(version int, contextID string, tcpPorts, udpPorts
 		zap.L().Error("unable to create cleanup configuration", zap.Error(err))
 		return err
 	}
-
+	markIntVal, _ := strconv.Atoi(mark)
```
Comment (mheese) [inline controller/internal/supervisor/iptablesctrl/iptables.go]: this function _has_ an error signature ...

Comment (amitlimaye) [reply controller/internal/supervisor/iptablesctrl/iptables.go]: fixed

File: controller/internal/supervisor/iptablesctrl/templates.go [hunk 2]
```diff
@@ -244,14 +258,22 @@ func (i *iptables) newACLInfo(version int, contextID string, p *policy.PUInfo, p
...
 		QueueBalanceNetSynAck:   i.fqc.GetNetworkQueueSynAckStr(),
 		QueueBalanceNetAck:      i.fqc.GetNetworkQueueAckStr(),
-		InitialMarkVal:          strconv.Itoa(cgnetcls.Initialmarkval - 1),
+		NetSynQueues:            i.fqc.NetworkSynQueues,
+		NetAckQueues:            i.fqc.NetworkAckQueues,
+		NetSynAckQueues:         i.fqc.NetworkSynAckQueues,
+		AppSynQueues:            i.fqc.ApplicationSynQueues,
+		AppSynAckQueues:         i.fqc.ApplicationSynAckQueues,
+		AppAckQueues:            i.fqc.ApplicationAckQueues,
+		InitialMarkVal:          strconv.Itoa((constants.Initialmarkval - 1) << constants.MarkShift),
```
Comment (mheese) [inline controller/internal/supervisor/iptablesctrl/templates.go:267]: stupid question probably: why do we need the -1 here before the shift?

**Review by mheese — DISMISSED** [review]
looks good to me! ... took quite some time to review, if somebody else from the enforcer team could also review this, that would give me some more peace of mind

**Pull request merged**